templates/default/filebeat-serving.yml.erb

```diff
@@ -0,0 +1,123 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: filebeat-config
+  namespace: hops-system
+  labels:
+    app: filebeat
+data:
+  filebeat.yml: |-
+    filebeat.autodiscover:
+      providers:
+        - type: kubernetes
+          node: ${NODE_NAME}
+          templates:
+            - condition.has_fields: ['kubernetes.labels.serving_hops_works/id']
+              config:
+                - type: container
+                  fields:
+                    source: "kube"  
+                  paths:
+                    - '/var/lib/docker/containers/${data.kubernetes.container.id}/*.log'
+                  exclude_lines: ["^\\s+[\\-`('.|_]"]  # drop asciiart lines
+    output:
+      logstash:
+        hosts: ["<%= @logstash_endpoint %>"]
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata: 
+  name: filebeat
+  namespace: hops-system
+  labels:
+    app: filebeat
+spec:
+  selector:
+    matchLabels:
+      app: filebeat
+  template:
+    metadata:
+      labels:
+        app: filebeat
+    spec:
+      serviceAccountName: filebeat
+      terminationGracePeriodSeconds: 30
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: filebeat
+          image: docker.elastic.co/beats/filebeat:<%= node['kube-hops']['filebeat']['version'] %>
```

Comment:
No you can't do that. 1st because of licensing and 2nd because we need airgapped deployments to work. 
You need to build a Docker image including filebeat from opendistro and then publish it as a tar file and upload it to the registry.

I think there was a comment with the same recommendation also in the previous PR.

Reply:
I'll upgrade filebeat in docker-images repo to 7.9.3 and use that one.